cmd/manager/main.go

```diff
@@ -85,12 +85,13 @@ func run() error {
 		return err
 	}
 
-	// Create a new Cmd to provide shared dependencies and start components
-	mgr, err := manager.New(cfg, manager.Options{
+	options := manager.Options{
 		Namespace:          "", //watch all namespaces
-		MapperProvider:     restmapper.NewDynamicRESTMapper,
 		MetricsBindAddress: fmt.Sprintf("%s:%d", metricsHost, metricsPort),
-	})
+	}
+
+	mgr, err := manager.New(cfg, options)
+
```

Comment:
extra line

Reply:
not sure if before or after but added a commit